app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/passingRateSummaryControls.jsx

```diff
@@ -35,6 +40,18 @@ const messages = defineMessages({
     id: 'PassingRateSummaryControls.ItemsValidationError',
     defaultMessage: 'Items count should have value from 1 to 600',
   },
+  PassingRateOptionTotal: {
+    id: 'PassingRatePerLaunchControls.PassingRateOptionTotal',
+    defaultMessage: 'Total test cases (Passed, Failed, Skipped)',
+  },
+  PassingRateOptionExcludingSkipped: {
+    id: 'PassingRatePerLaunchControls.PassingRateExcludingSkipped',
+    defaultMessage: 'Total test cases excluding Skipped',
+  },
+  PassingRateFormGroupControlLabel: {
+    id: 'PassingRatePerLaunchControls.PassingRateFormGroupControlLabel',
```

Comment:
It will lead to duplicating messages in translated files after regeneration.
Could you move the common messages for these two widgets to `passingRateChart`?

Reply:
May be it was good idea, to keep this messages in RadioGroupControl component?